cognite/client/_api/datapoints.py

```diff
@@ -1271,23 +1277,37 @@ def retrieve_latest(
                 >>> latest_abc = res[0][0]
                 >>> latest_def = res[1][0]
 
-            If you need to specify a different value of 'before' for each time series, or different values for
-            unit or unit system, you may pass several LatestDatapointQuery objects::
+            If you for example need to specify a different value of 'before' for each time series, you may pass several
+            LatestDatapointQuery objects. These will override any parameter passed directly to the function and also allows
+            for individual customisation of 'target_unit', 'target_unit_system', 'include_status', 'ignore_bad_datapoints'
+            and 'treat_uncertain_as_bad'.
 
                 >>> from datetime import datetime, timezone
                 >>> id_queries = [
                 ...     123,
                 ...     LatestDatapointQuery(id=456, before="1w-ago"),
                 ...     LatestDatapointQuery(id=789, before=datetime(2018,1,1, tzinfo=timezone.utc)),
                 ...     LatestDatapointQuery(id=987, target_unit="temperature:deg_f")]
+                >>> xid_queries = [
```

Comment:
super optional nit: It took me a second to connect that `xid` means `external_id` in this context. Maybe rename to `external_id_queries`?

Reply:
Sure, as an example I think clarity is essential 👌